diff -up usr/lib/python2.7/site-packages/pyanaconda/users.py.orig usr/lib/python2.7/site-packages/pyanaconda/users.py
--- usr/lib/python2.7/site-packages/pyanaconda/users.py.orig	2013-07-17 14:55:24.851345581 +0200
+++ usr/lib/python2.7/site-packages/pyanaconda/users.py	2013-07-17 14:56:24.086773290 +0200
@@ -313,6 +313,8 @@ class Users:
                     else:
                         password = cryptPassword(kwargs["password"], algo=kwargs.get("algo", None))
                     self.admin.setpassUser(userEnt, password, True)
+                    userEnt.set(libuser.SHADOWLASTCHANGE, "")
+                    self.admin.modifyUser(userEnt)
                 elif pw == "":
                     # Setup the account with *NO* password
                     self.admin.unlockUser(userEnt)
@@ -385,6 +387,7 @@ class Users:
         if lock:
             self.admin.lockUser(user)
 
+        user.set(libuser.SHADOWLASTCHANGE, "")
         self.admin.modifyUser(user)
 
     def setRootPassword(self, password, isCrypted=False, isLocked=False, algo=None):
